frontend/src/views/camp/Dashboard.vue

```diff
@@ -24,6 +24,10 @@
         :loading-endpoints="loadingEndpoints"
         :camp="camp"
         :periods="periods"
+        :filter-fn="
+          (filter) =>
+            scheduleEntries.filter((scheduleEntry) => filterFn(scheduleEntry, filter))
+        "
```

Comment:
I'm not sure if this should be a computed function / or if this is automatically made computed

Reply:
[2821b8f](https://github.com/ecamp/ecamp3/pull/7376/commits/2821b8ff4b258a60d982cf76ce368738e248877a)